### PR TITLE
node.js: changes in streaming

### PR DESCRIPTION
**Pull request by etimr**

### CODE REVIEW

**Review by vkozyura — APPROVED** [review]

**Comment by vkozyura** [comment]
@renejeglinsky, @smahati: Hi, Please review. The corresponding features are already released. Best regards, Vitaly

**Review by johannes-vogel — APPROVED** [review]

File: guides/databases-sqlite.md [hunk 1]
```diff
@@ -692,36 +692,6 @@ Generic application service handlers use *SELECT.localized* to request localized
...
 
-
-
-
-
-<!-- ### New Streaming API {.impl .node}
-
-TODO: New STREAM event, ...
-
--->
```
Comment (renejeglinsky) [inline guides/databases-sqlite.md:703]: Is this removed on purpose?

File: guides/databases.md [hunk 2]
```diff
@@ -764,3 +764,22 @@ In case of conflicts, follow these steps to provide different models for differe
 
 CAP samples demonstrate this in [cap/samples/fiori](https://github.com/SAP-samples/cloud-cap-samples/commit/65c8c82f745e0097fab6ca8164a2ede8400da803). <br>
 There's also a [code tour](https://github.com/SAP-samples/cloud-cap-samples#code-tours) available for that.
+
+## Node.js Runtime Handling BLOBs
```
Comment (renejeglinsky) [inline guides/databases.md:768]: I don't understand, why that has been moved here. When looking at the structure it feels like it doesn't belong. Maybe it's just the title? How about removing the "Node.js Runtime" and add that to the text? How does Java do that?
image

File: guides/media-data.md [hunk 3]
```diff
@@ -187,7 +187,8 @@ GET $filter=ID eq binary'Q0FQIE5vZGUuanM='
 
 - The usage of binary data in some advanced constructs like the `$apply` query option and `/any()` might be limited.
 - On SQLite, binary strings are stored as plain strings, whereas a buffer is stored as binary data. As a result, if in
-a CDS query, a binary string is used to query data stored as binary, this wouldn't work.
+a CDS query, a binary string is used to query data stored as binary, this wouldn't work. 
+- Please note, that SQLite doesn't support streaming. That means, that LargeBinary fields are read as a whole (not in chunks) and stored in memory, which can impact performance.
```
Comment (renejeglinsky) [inline guides/media-data.md:191]:
```suggestion
- Please note, that SQLite doesn't support streaming. That means, that `LargeBinary` fields are read as a whole (not in chunks) and stored in memory, which can impact performance.
```

File: node.js/core-services.md [hunk 4]
```diff
@@ -1084,7 +1086,9 @@ const stream = await srv.stream( SELECT('image').from('Foo',111) )
 stream.pipe(process.stdout)
 ```
 
-
+::: warning
+This API is limited to [database services](databases).
```
Comment (renejeglinsky) [inline node.js/core-services.md:1090]: Is there a more specific section we can link to? If not, maybe we can create one? Probably using the content that has been moved to this guide as a starting point for this new section (see my previous comment)?

File: guides/databases-sqlite.md (hunk 1 above)
Comment (johannes-vogel) [reply guides/databases-sqlite.md:703]: yes that was experimental and was completely removed now.

File: guides/databases.md (hunk 2 above)
Comment (johannes-vogel) [reply guides/databases.md:768]: It was moved here because it's not sqlite specific but node runtime specific.

Comment (renejeglinsky) [reply guides/databases.md:768]: How is this related to https://cap.cloud.sap/docs/node.js/best-practices#custom-streaming-beta ?

**Review by renejeglinsky — APPROVED** [review]

**Pull request merged**